utils/rest_api/create_task.py

```diff
@@ -0,0 +1,259 @@
+    assert len(jobs) == 1
+    job_id = jobs[0]["id"]
+    url_for_job = cvat_uri + "/?id={}".format(job_id)
+    print("url_for_job =", url_for_job)
+    res["url_for_job"] = url_for_job
+    res["status"] = True
+    return res
+
+def print_table_for_jira(list_all_done_res):
+    print_hrule("CURRENT RESULTS:")
+    for res in list_all_done_res:
+        assert res["status"]
+        print("|{}|[{}]|".format(res["full_relative_path"], res["url_for_job"]))
+    print_hrule("END OF CURRENT RESULTS")
+
+def main():
+    epilog_help_string = ("""
+    Please, note that argument path_json_labels_descr should point to a json file that describes labels for a task.
+    Example 1:
+        [
+            {
+                "name": "obj",
+                "attributes": []
+            }
+        ]
+    -- this will create for the task one bbox property with name 'obj' without attributes,
+    it is equivalent to CVAT label string 'obj'
+
+    Example 2:
+        [
+            {
+                "name": "obj",
+                "attributes": [
+                                   {
+                                      "name": "class",
+                                      "mutable": false,
+                                      "input_type": "text",
+                                      "default_value": "",
+                                      "values": [""]
+                                   }
+                ]
+            }
+        ]
+    -- this will create for the task one bbox property with name 'obj' with one non-mutable text attribute 'class',
+    it is equivalent to CVAT label string'obj @text=class:""'
+    """)
+
+    parser = ArgumentParser(description = "Script that receives list of videos and to creates CVAT tasks (one for the video), "
+            "upload the video to the task from the share, and print the table of created tasks that may be inserted into a JIRA ticket.",
+            epilog=epilog_help_string,
+            formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('--password_file', help='Path to a password file: text file with one line in format <username>:<password>')
+    parser.add_argument('--filelist', required=True, help='Path to a text file each line of which is a path to a video to create task. The videos should be on the file share connected to CVAT, the paths should be relative to the root folder of the CVAT file share. The name of each created task will be equal to the corresponding relative path.')
+    parser.add_argument('--bug_tracker', required=True, help='URI of JIRA ticket.')
+    parser.add_argument("-v", "--verbose", action="store_true", help="If HTTP requests should be logged.")
+    parser.add_argument("--path_json_labels_descr", required=True, help="Path to a json file that contains structure that should be used as description of labels for the task.")
+    parser.add_argument("--cvat_root_uri", required=True, help="URI of the CVAT server (without 'https://' prefix).")
```

Comment:
`--cvat_root_uri` --> `--server-host` and `--server-port` (default values should be localhost and 8080)

Reply:
Rename to `--cvat-server` with default value `localhost:8080`